cups.yaml

```diff
@@ -0,0 +1,119 @@
+package:
+  name: cups
+  version: 2.4.2
+  epoch: 0
+  description: The CUPS Printing System
+  target-architecture:
+    - all
+  copyright:
+    - paths:
+        - "*"
+      attestation: TODO
+      license: GPL-2.0-only
+
+secfixes:
+  2.4.2-r0:
+    - CVE-2022-26691
+
+environment:
+  contents:
+    repositories:
+      - https://packages.wolfi.dev/os
+    keyring:
+      - https://packages.wolfi.dev/os/wolfi-signing.rsa.pub
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - automake
+      - autoconf
+      - zlib-dev
+      - libpaper-dev
+      - dbus
+      - dbus-dev
+      - libusb-dev
+      - libgcrypt-dev
+
+      # - poppler-utils do we need this?
+      # - libjpeg-turbo-dev do we need this?
+      # - avahi-dev do we need this?
+      # - gnutls-dev do we need this?
```

Comment:
Not sure about these four packages.  Compiling seems fine as well as downstream packages that uses this.

If we don't need these I'll remove them from this PR or add a TODO.  If we do need these I'll need to create them as I've not done that yet.

Reply:
We will need them for CUPS to be actually useful for printing.

Reply:
ok cool - I'll do those too

Reply:
for now we could unblock JDK and address TODOs in a followup https://github.com/wolfi-dev/os/issues/49